nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -266,6 +266,9 @@ class VanillaDataManagerConfig(InstantiateConfig):
     camera_optimizer: CameraOptimizerConfig = CameraOptimizerConfig()
     """Specifies the camera pose optimizer used during training. Helpful if poses are noisy, such as for data from
     Record3D."""
+    use_equirect_sampler: bool = False
```

Comment:
I need to think a bit more, but is there any way to infer whether to use this sampler given the camera_type?

Reply:
self.train_dataset.dataparser_outputs.cameras.camera_type[0] == CameraType.EQUIRECTANGULAR.value

Reply:
How should we handle mixed camera types? e.g. some images equirect, some perspective. I don't think its possible at the moment, but the `Cameras` data structure allows for the possibility

Reply:
I see this disclaimer: https://github.com/nerfstudio-project/nerfstudio/blob/2c64ba5b6aa7c47f2e214572ad4d8c8759888b20/nerfstudio/cameras/cameras.py#L54-L56

If its OK for now to assume that all cameras in a `Cameras` take the same type, then I can do a bunch of simplifications in the pixel sampler

Reply:
You could fall back to using the normal sampler if any camera isn't equirectangular, (and maybe print a warning).

Reply:
I am fine with this